src/sensors.jl

```diff
@@ -18,151 +18,145 @@ module Sensors
         Lib.Sensors_ResetAll()
     end
 
-    """(read-only) Array of Sensor names."""
+    """Array of Sensor names."""
     function AllNames()::Vector{String}
         return Utils.get_string_array(Lib.Sensors_Get_AllNames)
     end
 
-    """(read-only) Number of Sensors in Active Circuit."""
+    """Number of Sensors in Active Circuit."""
     function Count()::Int
         return Lib.Sensors_Get_Count()
     end
 
-    """Array of doubles for the line current measurements; don't use with kWS and kVARS."""
+    """Array of doubles for the line current measurements; don't use with kWS and kVARS. (Getter)"""
```

Comment:
What do "kWS" and "kVARS" mean here?

If those are power measurements, I'd use "kW" and "kvar" with no "S" at the end.

Reply:
Fixed in cc7d20d.